WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m

```diff
@@ -204,6 +204,10 @@ - (void)trackMixpanelDataForInstructions:(WPAnalyticsTrackerMixpanelInstructions
     [instructions.peoplePropertiesToAssign enumerateKeysAndObjectsUsingBlock:^(id key, id obj, BOOL *stop) {
         [self setValue:obj forPeopleProperty:key];
     }];
+    
+    [instructions.superPropertiesToAssign enumerateKeysAndObjectsUsingBlock:^(id key, id obj, BOOL *stop) {
```

Comment:
I'm aware we want to change this in the future, but for now let's hold off on it until the next release as I was trying to keep this change more minimal given how late we are in the cycle.